definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2992,5 +2992,87 @@
     "PostConditionState": "*",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/return-to-state-before-general-referral",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "solicitorStatementOfTruthPaySubmit",
+    "Name": "Case submission",
+    "Description": "Agree Statement of Truth, Pay & Submit",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "SOTAgreementPayAndSubmitRequired",
+    "PostConditionState": "*",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/petition-issue-fees",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/process-pba-payment",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "EndButtonLabel": "Submit Petition"
+  },
+  {
+    "LiveFrom": "01/04/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "serveByAlternativeMethod",
+    "Name": "Serve by alternative method",
+    "Description": "Serve by alternative method",
+    "DisplayOrder": "19",
+    "PreConditionState(s)": "GeneralConsiderationComplete;Issued",
+    "PostConditionState": "AwaitingAlternativeService",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "reissueFromRejected",
```

Comment:
these are being moved from prod only back to base file, why were these in prod only?

Reply:
because they were existing events that we moved from the core file to prod and nonprod, so that we could have update nonprod version.

We can't define the same event twice (in the core and nonprod). It would fail.

Reply:
but the `reissueFromRejected` events weren't in the nonprod file

Reply:
as long as there's no reason for them not to be in nonprod config then I've no issues with this change

Reply:
Apparently somebody made a mistake and move an event that didn't need nonprod version.

It was in prod file: https://github.com/hmcts/div-ccd-definitions/pull/809/files#diff-f7d2648557ca22fce8362231d15c91c1569efa6c5c500b9e7191742e34dfe5fbL5

Please mind there was over 20 PRs for Share a Case and some original decisions have changed few times.